build-capnp.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Syntax: build-capnp.sh TARGET
+#     TARGET = darwin or linux
+# 
+# This script downloads the capnproto source distribution, unpacks it to a `./build-capnp`
+# subdirectory, and cross-compiles its libraries and headers from a linux host to the specified
+# target. We produce static libraries for the `linux` target (to keep distribution simple), but
+# dynamic libraries for `darwin`, because the Mac system libraries have a tendency to break.
+# 
+# For the `linux` target, the host must have `clang` installed and available on their PATH. The
+# `darwin` target is based on the `osxcross` toolchain, which provides `o64-clang++`.
+# 
+# Copyright Fourier Audio Ltd. 2022. All Rights Reserved.
+
+# See: https://sipb.mit.edu/doc/safe-shell/
+set -euf -o pipefail
+
+# Check argument count
+if [[ $# != 1 ]]; then
+    echo "Expected exactly one argument, but received ${#}."
+    exit 1
+fi
+
+# Get directory of this script in a relatively robust fashion (pun intended); see
+# http://www.binaryphile.com/bash/2020/01/12/determining-the-location-of-your-script-in-bash.html
+here=$(cd "$(dirname "$BASH_SOURCE")"; \
+    cd -P "$(dirname "$(readlink "$BASH_SOURCE" || echo -e "$BASH_SOURCE")")"; \
+    pwd)
+
+# We store all of our build artefacts in a subdirectory. If that directory already exists, remove
+# it so that we can cleanly restart from scratch.
+build_dir="${here}/build-capnp"
+rm -rf ${build_dir}
+mkdir -p ${build_dir}
+
+pushd ${build_dir}
+
+# Download and unpack the capnproto source. It's pretty small (1.6 MiB), so this should be fast.
+curl -O https://capnproto.org/capnproto-c++-0.10.3.tar.gz
```

Comment:
Now that we are not using the system's libcapnp, `install=deps.sh` in this repository is unnecessary and should be removed.